maintainer/gh_post_status.sh

```diff
@@ -8,4 +8,4 @@ STATUS="$1"
 curl "https://api.github.com/repos/espressomd/espresso/statuses/$GIT_COMMIT?access_token=$GITHUB_TOKEN" \
      -H "Content-Type: application/json" \
      -X POST \
-     -d "{\"state\": \"$STATUS\", \"context\": \"ICP GitLab CI\", \"target_url\": \"$URL\"}"
+     -d "{\"state\": \"$STATUS\", \"context\": \"ICP GitLab CI\", \"target_url\": \"$URL\", \"description\": \"GitLab CI is performing tests.\"}"
```

Comment:
Depending on whether `$STATUS` is `pending`, `success` or `failure`, the description should be different.

Reply:
was a test, removed it